packages/netsuite-adapter/src/change_validators/role_permission_ids.ts

```diff
@@ -0,0 +1,74 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { ChangeError, getChangeData, InstanceElement, isInstanceChange } from '@salto-io/adapter-api'
+import { logger } from '@salto-io/logging'
+import { values } from '@salto-io/lowerdash'
+import { ROLE } from '../constants'
+import { NetsuiteChangeValidator } from './types'
+import { ID_TO_PERMISSION_INFO } from '../autogen/role_permissions/role_permissions'
+
+const { isDefined } = values
+const log = logger(module)
+
+type RolePermissionType = {
+  permkey: string
+  permlevel: string
+}
+
+const permissionsToAdd: Record<string, Set<string>> = {
+  // mapping of permissions which aren't included in the documentation table
+  LIST_BULK_PROCESSING: new Set(['VIEW']),
+  LIST_ENTITYSUBSIDIARYRELATION: new Set(['VIEW']),
+  LIST_DEPARTMENT: new Set(['VIEW']),
+}
+
+const isValidPermissions = (permkey: string, permlevel: string): boolean => {
```

Comment:
Since both input params are strings, this is safer:
```suggestion
const isValidPermissions = ({ permkey, permlevel }: RolePermissionType): boolean => {
```